src/xcode/ENA/ENA/Source/Workers/WebPageHelper.swift

```diff
@@ -37,7 +37,17 @@ enum WebPageHelper {
 		}
 	}
 
-	static func openSafari(withUrl url: URL, from viewController: UIViewController) {
+	static func open(withUrl url: URL, from viewController: UIViewController) {
+		switch url.scheme {
+		case "tel", "mailto":
+			UIApplication.shared.open(url, options: [:], completionHandler: nil)
+		default:
+			openLink(withUrl: url, from: viewController)
+		}
+
+	}
+
+	static func openLink(withUrl url: URL, from viewController: UIViewController) {
```

Comment:
To me it looks like the complete code in this method is a duplicated code from ```showWebPage``` method. Ideally, this should be refactored so that only one method exists, when this code is already touched.

Reply:
hm, that's interesting. Since i just moved the already existing code into another method, i believe this duplication was already there before i touched anything at all. But i agree with you, this is an opportunity to remove duplicate code, so i will go ahead and do that! 💯 👍

Reply:
(I moved around the parameters as well, because they were inconsistent with openLink.)